packages/zipkin-instrumentation-request/src/wrapRequest.js

```diff
@@ -0,0 +1,34 @@
+const {Request, Annotation} = require('zipkin');
+
+function wrapRequest(request, {tracer, serviceName = 'unknown', remoteServiceName}) {
+  return request.defaults((options, callback) => {
+    tracer.scoped(() => {
+      tracer.setId(tracer.createChildId());
+      const traceId = tracer.id;
```

Comment:
What's the recommended way to share trace id across request/response? Does it have to fall within a single `tracer.scoped(...)`? I see some use `this.traceId` (ex. cujojs) and then others use `const traceId` (ex. fetch).